app/views/teachers/RequestQuoteView.coffee

```diff
@@ -136,6 +137,11 @@ module.exports = class RequestQuoteView extends RootView
       window.tracker?.trackEvent 'Teachers Request Demo Form Started', category: 'Teachers', ['Mixpanel']
     @formChanged = true
 
+  onClickRequestButton: (e) ->
+    eventAction = $(e.target).data('event-action')
```

Comment:
Is it ever possible for `.data` to be undefined or not a function?

Reply:
`.data()` is a jquery function that stores the data associated with the matched element. So I believe it should not be undefined.